Makefile

```diff
@@ -26,6 +26,7 @@ MANAGER_IMAGE_PATCH := "apiVersion: apps/v1\
 \n    spec:\
 \n      containers:\
 \n      - image: <your image file>\
+\n        imagePullPolicy: IfNotPresent\
```

Comment:
Should not be `IfNotPresent`, as that would break using `:latest` to test.

I'd recommend making the policy`Always` if anything, to make sure people aren't testing their dev results with stale images.

Reply:
thanks for the review! that makes sense.

`imagePullPolicy` needs to be set to `IfNotPresent` for CI builds (this is getting loaded inside kind cluster using `kind load docker-image`) since these images are not pushed to any registry. Added a quick sed for inserting this value when running inside Travis only, let me know if this works.

Reply:
see failed test below: https://travis-ci.org/open-policy-agent/gatekeeper/builds/574525019